templates/hooks/table/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,21 @@
+	if tableDeleting(latest) {
+		msg := "table is currently being deleted"
```

Comment:
Move strings to constants in `hooks.go`?

Reply:
I'll leave it in the template file for now - just waiting to see what the generated `hooks.go`/`conditions.go` will look like in the future